### ISSUE TITLE
Craft 3.5 - `forms.checkbox` isn't parsing HTML tags

### ISSUE THREAD

**Issue by lindseydiloreto** [craftcms/cms#6673]
### Description

This unparsed HTML tag is now appearing on the **Star Ratings > Settings** page...

![settings@2x](https://user-images.githubusercontent.com/5309692/90591903-16622c80-e199-11ea-9847-a1457cd1db47.png)

Prior to Craft 3.5, the `<strong>` tag was getting correctly parsed into HTML. This no longer seems to be the case. I even tried replacing those tags with Markdown, but it had no affect... the Markdown wasn't parsed either.

That HTML is being generated by the following Twig code...

```twig
{{ forms.checkbox({
    label: "<strong>Require login to rate</strong>"|t,
    id: 'requireLogin',
    name: 'requireLogin',
    checked: settings.requireLogin,
    disabled: 'requireLogin' in overrideKeys
}) }}
```

^ This code hasn't changed in several years, so it's definitely due to a change on Craft's side. I'm not sure if the change was intentional (but I guess you'll let me know if it was). If that change _was_ intentional, how can I now work around it?

It's a pretty minor issue, but it's affecting several of my plugins (Upvote, Star Ratings, View Count, Digital Download).

Let me know what you think!

### Steps to reproduce

1. Install Craft 3.5.x
2. Install Star Ratings 2.x
3. Go to Star Ratings > Settings page
4. Witness unparsed HTML tags

### Additional info

- Craft version: 3.5.5
- Star Ratings version: 2.1.2

**Comment by brandonkelly** [comment]
Yes, this was an intended change in Craft 3.5. It was unavoidable to fix #5885 in a way that didn’t pose a security vulnerability. Per the release note, if you need to pass HTML into a checkbox label you can wrap it in the new `raw()` function:

```twig
{{ forms.checkbox({
    label: raw("<strong>Require login to rate</strong>"|t),
    id: 'requireLogin',
    name: 'requireLogin',
    checked: settings.requireLogin,
    disabled: 'requireLogin' in overrideKeys
}) }}
```

Note that you should only do this if you know the passed-in HTML can be trusted.

**Comment by lindseydiloreto** [comment]
Looks like that works perfectly. Thanks @brandonkelly! 🍺
